bindings/java/nz/mega/sdk/DelegateMegaTreeProcessor.java

```diff
@@ -1,5 +1,10 @@
 package nz.mega.sdk;
 
+/**
+ * Interface to process node trees.
+ * <p>
```

Comment:
Are these paragraph indicators (`<p>`) really required for rendering the JavaDoc output correctly? I think a blank line suffices (but I'm not sure). I think HTML markup is only needed for more detailed layout stuff (such as itemised or numbered lists).